modules/interacting-serverless-apps-http2-gRPC.adoc

```diff
@@ -76,3 +76,26 @@ grpc.Dial(
 ====
 Ensure that you append the respective port, 80 by default, to both hosts as shown in the previous example.
 ====
+
+. For {product-title} 4.10 and later, add the `serverless.openshift.io/default-enable-http2=true` annotation to the `KnativeServing` Custom Resource:
```

Comment:
Could you add this section as the different steps?
Current docs shows that it is as "step 4" but the steps are independent.

![image](https://user-images.githubusercontent.com/2138339/229732472-60271369-882e-4fff-9778-b2ffaec3bc92.png)

Reply:
Thank you @nak3.

To clarify, this is what you reviewed:

```
Procedure
---------

1. Find the application host...

2. Find the ingress gateway’s public address...

3. Manually set the host header... 

4. For OpenShift Dedicated 4.10 and later, add...

5. After the annotation is added, you can verify...
```


And you now suggest that I change it to this, is that correct?:

```
Procedure 1 (for 4.10+)
-----------------------

1. For OpenShift Dedicated 4.10 and later, add...

2. After the annotation is added, you can verify...

Procedure 2 (for up to 4.9)
---------------------------

1. Find the application host...

2. Find the ingress gateway’s public address...

3. Manually set the host header... 
```


However, didn't you say that the steps in Procedure 2 are prerequisite for steps in Procedure 1?

`"Enable HTTP/2 on OpenShift Ingress controller as per OCP documentation Enabling HTTP/2 Ingress connectivity"`

If there actually no such prerequisite, then your new suggested structure works. Otherwise, it is better to put Procedure 2 before Procedure 1 (and IMO it's better to keep them as a single procedure as it is now, unless I misunderstand something).

Reply:
Ah! Thank you for the confirmation. I noticed that I put a wrong link in the draft. Here is the correct [link](https://docs.openshift.com/container-platform/4.12/networking/ingress-operator.html#nw-http2-haproxy_configuring-ingress) .

So, the `Prerequisites` also different and it should be:

```
Prerequisites 1
---
* OpenShift Serverless Operator and Knative Serving are installed on your cluster.
* Install the OpenShift CLI (oc).
* You have created a Knative service.
* OpenShift 4.10 or later.
* Enable HTTP/2 on OpenShift Ingress controller as per OCP documentation [Enabling HTTP/2 Ingress connectivity](
https://docs.openshift.com/container-platform/4.12/networking/ingress-operator.html#nw-http2-haproxy_configuring-ingress)

Procedure 1 (for 4.10+)
-----------------------

1. For OpenShift Dedicated 4.10 and later, add...

2. After the annotation is added, you can verify...


Prerequisites 2
---
* OpenShift Serverless Operator and Knative Serving are installed on your cluster.
* Install the OpenShift CLI (oc).
* You have created a Knative service.

Procedure 2 (for up to 4.9)
---------------------------

1. Find the application host...

2. Find the ingress gateway’s public address...

3. Manually set the host header...
```